twitter4j-core/src/main/java/twitter4j/conf/ConfigurationBase.java

```diff
@@ -73,6 +73,7 @@
     private boolean includeEntitiesEnabled = true;
     private boolean trimUserEnabled = false;
     private boolean includeExtAltTextEnabled = true;
+    private boolean tweetModeExtended = false;
```

Comment:
I would favor this being a String or Enum value to accomodate any future usage of this tweet_mode parameter by Twitter with new values.